pkg/policy/policy.go

```diff
@@ -78,21 +95,50 @@ func (pp Policies) All() []string {
 	return policyNames
 }
 
-func (pp Policies) Properties() map[string]interface{} {
-	usedCustom := 0
-	policies := make([]*Meta, 0)
+func (pp Policies) Get(policyName, subPath string) Policies {
+	if subPath == "" {
+		policyName, subPath = getter.ParseSourceSubPolicy(policyName)
+	}
+
 	for _, p := range pp {
-		if p.meta == nil || p.meta.Type != "hub" {
-			usedCustom++
-			// we don't add info about custom policies that were executed
+		if policyName != p.Name {
 			continue
 		}
-		policies = append(policies, p.meta)
+		if subPath != "" && p.SubPolicy() == "" {
+			return Policies{
+				&Policy{
+					Name:   p.Name,
+					Source: p.Source + "//" + subPath,
+				},
+			}
+		}
+		return Policies{p}
 	}
-	return map[string]interface{}{
-		"used_custom": usedCustom,
-		"policies":    policies,
+	return nil
+}
+
+func (p Policy) Analytic(dbPersistence, usedConfig bool) Analytic {
+	pa := Analytic{
+		Persistence: dbPersistence,
+		UsedConfig:  usedConfig,
+		Name:        p.Name,
+		Type:        p.SourceType(),
+		Selector:    p.SubPolicy(),
+		Private:     p.SourceType() != "hub",
```

Comment:
[non blocking] Could we remove `Private`? Seems like `Type` has that information already

Reply:
Its more for analytics ease of use